src/triage/experiments/base.py

```diff
@@ -222,6 +228,28 @@ def initialize_components(self):
                 "you will not be able to make matrices."
             )
 
+        if "bias_audit_config" in self.config:
+            bias_config = self.config["bias_audit_config"]
+            self.protected_groups_table_name = "protected_groups_{}".format(
```

Comment:
You could get some more reuse out of this if you namespaced it with something more granular than the experiment hash.  We can hash arbitrary things with `filename_friendly_hash`, so you could hash the bias config and use that. However, there is the added complication of the cohort.  There are two approaches I see:

1. Treat the protected groups table as being persistent across cohorts, so data from different cohorts live alongside each other in the table, with a cohort hash column to disambiguiate.  We already do hash the cohort config in https://github.com/dssg/triage/blob/master/src/triage/experiments/base.py#L183-L186 , you could refactor this out to send the hash to the ProtectedGroupsGenerator as well.

2. Treat the protected groups table as only being true for one specific cohort, and include the cohort hash in the name of the table along with the protected groups hash (e.g. `protected_groups_abcdef_cohort_ghijkl`)

I like #1 a bit better, in part because not requiring the cohort to be represented in the table name would open up something more readable, such as a user-configured protected group config name. So if they added a name called 'hispanic', the name of the table would end up being `protected_groups_hispanic_abcdef`, which is easier to find later than something just named with a hash.